website/docs/docs/cloud/configure-cloud-cli.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Configure dbt Cloud CLI
+id: configure-cloud-cli
+description: "Instructions on how to configure the dbt Cloud CLI"
+---
+
+:::info Public preview functionality
+
+The dbt Cloud CLI is currently in [public preview](/docs/dbt-versions/product-lifecycles#dbt-cloud). Share feedback or request features you'd like to see on the [dbt community Slack](https://getdbt.slack.com/archives/C05M77P54FL).
+
+::: 
+
+
+## Prerequisites
+
+- You must set up a project in dbt Cloud.
+- You must have your [personal development credentials](/docs/dbt-cloud-environments#set-developer-credentials) set for that project. The dbt Cloud CLI will use these credentials, stored securely in dbt Cloud, to communicate with your data platform.
+- You must [enroll](/docs/dbt-versions/experimental-features) in the dbt Cloud beta features. 
+	- To enroll, navigate to your **Profile Settings** and enable the **Beta** flag under **Experimental Features**.
+
+
+## Configure the dbt Cloud CLI
+
+Once you install the dbt Cloud CLI, you need to configure it to connect to a dbt Cloud project.
+
+1. Ensure you meet the prerequisites above.
+2. Create an environment variable with your [dbt Cloud API key](/docs/dbt-cloud-apis/user-tokens):
+   - On MacOS, Linux, or Windows add an environment variable:
+
+        ```bash
+        export DBT_CLOUD_API_KEY="1234" # Replace 1234 with your API key   
+        ```
+
```

Comment:
Form line 34 down the text turns blue in GitHub. I think this means there's a space or code error before it. It looks fine in the output, so maybe nbd

Reply:
thank you Leona! looks like Greg made some updates